packages/ui/src/services/api-docs/api-docs-helpers.ts

```diff
@@ -79,31 +80,31 @@ export const getApiTableData = (
 
   // Events
   for (const eventName in manualApiOptions.events) {
-    const manualEventOptions: ManualEventApiOptions = manualApiOptions.events[eventName] || {}
+    const manualEventOptions: ManualEventApiOptions = manualApiOptions.events[camelCase(eventName)] || {}
```

Comment:
Not needed. camelCase is mandatory for api config.

Reply:
![image](https://user-images.githubusercontent.com/29925191/81100565-6af8d900-8f15-11ea-8e70-2201591297fb.png)
without camelCase

Reply:
How did you get non camelCase prop in the first place?

Reply:
camelCase and kebabCase are the same if you have 1 word